docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 <!-- Thank you for submitting this PR. You are awesome!
 -->
 
-**Status**:
-- [ ] Ready
-- [ ] In Development
-- [ ] On Hold
+**Checklist**
+
+- [ ] My branch is up-to-date with the Upstream `development` branch.
```

Comment:
The upstream branch is called `develop` instead of `development`.

Also, maybe `upstream` instead of `Upstream`? No strong opinions on this.